docs/documentations/gestor-de-dependencias.md

```diff
@@ -0,0 +1,13 @@
+De cara a la elección del gestor de dependencias, estas son algunas de las posibilidades:
```

Comment:
Ya lo puse el jueves/viernes pasado, pero se ve que hay que [deciros personalmente que consultéis los errores frecuentes](https://github.com/JJ/IV-22-23/blob/master/sesiones/semana-08.md#objetivo-3).